submissions/shawn-sean-anthony/scripts/article.js

```diff
@@ -0,0 +1,60 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+
```

Comment:
As soon as we clone the article with a class of template, we can save this into a new jQuery object, referenced as $newArticle, which now gives us all of the methods we need to use below.